automation/jleechanorg_pr_automation/tests/test_codex_cli_tasks.py

```
@@ -0,0 +1,583 @@
+            "is_review": False,
+            "attempt_total": 1,
+        },
+    ],
+    "cursor": "some-cursor-token",
+}
+class TestFilterGithubMentions(unittest.TestCase):
+    """filter_github_mentions returns only GitHub Mention-prefixed tasks."""
+
+    def test_filters_correctly(self):
+        result = CodexCloudAPI.filter_github_mentions(SAMPLE_LIST_RESPONSE["tasks"])
+        self.assertEqual(len(result), 2)
```

Ruff (0.14.14)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)